recipes/pystackreg/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "pystackreg" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "2bc8bdac892bb827cd2725b8ee45bbcdb8835f2fb1d41863f6d50c77f7e36ccc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy 1.16
```

In the feedstock this should be changed back to
```suggestion
    - numpy
```
and then be rerendered.

---

Oh, ups. Did not realize it should not go here. Will fix it.